lib/temporal/client.rb

```diff
@@ -172,6 +172,27 @@ def signal_workflow(workflow, signal, workflow_id, run_id, input = nil, namespac
       )
     end
 
+    # Issue a query against a running workflow
+    #
+    # @param workflow [Temporal::Workflow, nil] workflow class or nil
```

Comment:
If this arg can be nil, would be nice to see a test that confirms the logic still works. Presumably only `workflow_id` must be populated for this to match. Can/should it raise if that arg is missing?